website/static/_redirects

```diff
@@ -15,6 +15,10 @@
 # Changed blog post dates
 /blog/2021/03/11/version-0.64   /blog/2021/03/12/version-0.64
 
+# Removed pages
+/docs/new-architecture-app-modules-ios.md       /docs/new-architecture-intro.md
+/docs/new-architecture-app-renderer-ios.md      /docs/new-architecture-intro.md
```

Comment:
```suggestion
/docs/new-architecture-app-modules-ios            /docs/new-architecture-intro
/docs/new-architecture-app-renderer-ios           /docs/new-architecture-intro
/docs/next/new-architecture-app-modules-ios       /docs/next/new-architecture-intro
/docs/next/new-architecture-app-renderer-ios      /docs/next/new-architecture-intro
```

Small update for the redirect, since they don't work on the preview deploy.